docs/IR.md

```diff
@@ -335,6 +335,7 @@ The following types are used to define the types of graph and node inputs and ou
 ONNX|dense tensors|Tensors are a generalization of vectors and matrices; whereas vectors have one dimension, and matrices two, tensors can have any number of dimensions, including zero. A zero-dimensional tensor is logically equivalent to a scalar value.
 ONNX|sequence|Sequences are dense, ordered, collections of elements that are of homogeneous types.
 ONNX|map|Maps are associative tables, defined by a key type and a value type.
+ONNX|optional|Optionals are a wrappers that may contain an element of tensor, sequence, or map type, or may be empty (containing none).
 
 ONNX currently does not define a sparse tensor type.
```

Comment:
May need to update above line

Reply:
Can we delete the above line "ONNX currently does not define a sparse tensor type." Thanks.